batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.lftechnology.batch7crud.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/19/16
+ */
+public abstract class CustomHttpServlet extends HttpServlet {
+  private static final String ERROR_MESSAGE = "errorMessage";
+
+  protected void showPageNotFound(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+    req.getServletContext().getRequestDispatcher("/WEB-INF/views/error.jsp").forward(req, resp);
+  }
+
+  protected void showServerError(HttpServletRequest req, HttpServletResponse resp, Throwable e) throws ServletException, IOException {
+    resp.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+    req.setAttribute(ERROR_MESSAGE, e.getMessage());
+    req.getServletContext().getRequestDispatcher("/WEB-INF/views/error.jsp").forward(req, resp);
```

Comment:
you are hard coding "/WEB-INF/views/error.jsp" as a jsp path. Does it work for windows system ? Think like how your code works regardless of any platform type.